docs/wiki/The CI environment - Publishing.md

```diff
@@ -20,6 +21,46 @@ Besides the publishing phase's runtime, there is also the possibility to set the
 > **Note**<br>
 > The `version` used for publishing any artifact is the same for all three target locations, which reduces the maintenance effort.
 
+# Considerations
+
+While the CARML CI-environment can publish to all locations mentioned above, you may effectively decide to configure only subset as per your requirements. To help you with the decision, the following content provides you with an overview of the possibilities of each target location.
+
+## Outline
+- **Template Specs**<p>
+  A [Template Spec](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-specs?tabs=azure-powershell) is an Azure resource with the purpose of storing & referencing Azure Resource Manager (ARM) templates. <p>
```

Comment:
I'd mention the template specs name length limitation

Reply:
added